### DISCUSSION TITLE
Question about doing 'npm i --package-lock-only' in yarn

### DISCUSSION THREAD

**Discussion by Seiji92** [npm/feedback#627] [CLI] [answered]
I have a reason to want to use 'npm i --package-lock-only' in a yarn builds to examine dependencies. Would using the command give me a proper package-lock.json or would I be missing anything?

**Comment by MylesBorins** [comment, marked as answer]
unfortunately we cannot offer support for yarn in this channel, which is primarily for making feature requests. I advise asking over at the yarn support channels.

**Reply by arcanis** [reply]
I think they'd like to generate a npm lockfile from a Yarn one, which isn't something we can help with.

**Reply by MylesBorins** [reply]
Sorry for passing the buck then. The best I can suggest would be to run `npm ls` to verify the package.json and lock match

With that said, this feedback repository is meant for feature requests not for general support. You may want to try opening an issue on stack overflow or github.community
